internal/cmd/config/config.go

```diff
@@ -379,6 +383,36 @@ func Parse(d string) (*Config, error) {
 						reflect.TypeOf(t).String())
 				}
 			}
+			if result.Controller.Database.MaxIdleConnectionsRaw != nil {
+				switch t := result.Controller.Database.MaxIdleConnectionsRaw.(type) {
+				case string:
+					maxIdleConnectionsString, err := parseutil.ParsePath(t)
+					if err != nil {
+						return nil, fmt.Errorf("Error parsing database max idle connections: %w", err)
+					}
+					result.Controller.Database.MaxIdleConnections, err = strconv.Atoi(maxIdleConnectionsString)
+					if err != nil {
+						return nil, fmt.Errorf("Database max idle connections value is not an int: %w", err)
+					}
+				case int:
+					result.Controller.Database.MaxIdleConnections = t
+				default:
+					return nil, fmt.Errorf("Database max idle connections: unsupported type %q",
+						reflect.TypeOf(t).String())
```

Comment:
Can we just use `%T`? https://pkg.go.dev/fmt#hdr-Printing

Reply:
I put this back after some tests failed expecting a quoted string around the type here. I think using `%q` looks nicer compared to `\"%T\"`